### DISCUSSION TITLE
ORB Feature Matching

### DISCUSSION THREAD

**Discussion by Leon-bro** [community/community#50293] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hello! I am doing diploma work, and took this algorithm for it. I have an improvement for BRIEF and corner orientation algorithm. It work faster, but doing the same things. It may be useful.
[utils.zip](https://github.com/community/community/files/10998573/utils.zip)

**Comment by jge162** [comment]
why don't you post on your GitHub and we can review it there. Doesn't seem safe to download a zip file.

**Comment by Leon-bro** [comment]
oh. i didnt know how to send it right. Okey i download it on my Git

**Reply by Leon-bro** [reply]
Here the code https://github.com/Leon-bro/ORB_Feature_Matching_Faster_Implementation.git
